### ISSUE TITLE
testing turtle lib in pyscript = fail

### ISSUE THREAD

**Issue by pleabargain** [pyscript/pyscript#218]
Not all libs will work. Turtle is apparently one of them

Here's the video
https://youtu.be/sqHN0XimukI

**Comment by JeffersGlass** [comment]
Correct, only [packages supported by Piodide](https://github.com/pyodide/pyodide/tree/main/packages) or pure Python packages that can be [installed via micropip](https://github.com/pyscript/pyscript/issues/85#issuecomment-1116838680) can be run imported, if I undrerstand right. Turtle (and tKinter which its based on) are both [not (yet) compatible with piodide's WASM VM](https://pyodide.org/en/stable/usage/wasm-constraints.html) and are not runnable at this time.

**Comment by rth** [comment]
See also https://stackoverflow.com/a/72100237/1791279

**Comment by pleabargain** [comment]
> 

Thank you for the detailed reply! I better understand the situation!
